src/ifd-nfc.c

```diff
@@ -444,8 +519,12 @@ IFDHGetCapabilities(DWORD Lun, DWORD Tag, PDWORD Length, PUCHAR Value)
       *Value  = 1;
       *Length = 1;
       break;
+#if defined(HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD) && HAVE_DECL_TAG_IFD_POLLING_THREAD
```

Comment:
I think you use ``#if`` to have the line of code:
```
case TAG_IFD_STOP_POLLING_THREAD:
```
used only if TAG_IFD_STOP_POLLING_THREAD is defined.

If TAG_IFD_STOP_POLLING_THREAD is defined and its value is 0 then the ``case TAG_IFD_STOP_POLLING_THREAD:`` is valid and should be used.

If that is not the case then please describe what you want the code to be in the following case:
#define HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD 0